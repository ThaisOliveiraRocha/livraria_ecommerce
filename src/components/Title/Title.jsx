import React from 'react';
import PropTypes from 'prop-types';
import { Titulo } from './Title.styles';

const Title = ({children, font, color, grow}) => (
  <Titulo font={font} color={color} grow={grow}>{children}</Titulo>
);

Title.propTypes = {
  // bla: PropTypes.string,
};

Title.defaultProps = {
  // bla: 'test',
};

export default Title;
