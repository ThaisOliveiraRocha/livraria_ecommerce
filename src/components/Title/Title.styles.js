import styled from "styled-components";

export const Titulo = styled.span`
  flex-grow: ${({ grow }) => (grow ? grow : 0)};
  font-size: ${({ font }) => (font ? font : "16px")};
  font-weight: 600;
  color: ${({ color }) => (color ? color : "#08234D")};
  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
`;
