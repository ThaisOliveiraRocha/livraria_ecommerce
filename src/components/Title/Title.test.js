import React from "react";
import { shallow } from "enzyme";
import Title from "./Title";

describe("<Title />", () => {
  it("should render without crashing", () => {
    shallow(<Title />);
  });
  it("should render basic infos", () => {
    const wrapper = shallow(<Title />);
    expect(wrapper).toMatchSnapshot();
  });
  
});
