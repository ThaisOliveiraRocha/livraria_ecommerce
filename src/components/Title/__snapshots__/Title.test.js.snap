// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render basic infos 1`] = `<styled.span />`;
